manifests/n/Nushell/Nushell/0.76.0/Nushell.Nushell.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with WinGet Releaser v2 using Komac v1.0.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Nushell.Nushell
+PackageVersion: 0.76.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- nu
+ReleaseDate: 2023-02-21
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/nushell/nushell/releases/download/0.76.0/nu-0.76.0-x86_64-pc-windows-msvc.msi
+  InstallerSha256: b010a33f982ca2685677122cad651101f8c93016f59f07f54c88ae2024da9c44

```

```suggestion
  InstallerSha256: B010A33F982CA2685677122CAD651101F8C93016F59F07F54C88AE2024DA9C44
```
Uppercase hash to match format of other manifests